spec/factories/old_customers.rb

```diff
@@ -0,0 +1,25 @@
+FactoryGirl.define do
+  factory :old_customer, :class => 'Customer' do
+    transient do
+      role :patron
+      created_by_admin false
+    end
+    sequence(:first_name) { |n| "Jack#{n}" }
```

Comment:
Faker gem already existing, recommend using that

Reply:
Jack wrote this, you didn't see it in his pull request? It's essentially copied from factories/customers.rb so it should be fine.

Reply:
Well, I didn't do code review for that one

Reply:
Since it is the same branch, it'd better be clean when it goes back